testing_and_setup/compass/ocean/jigsaw_to_MPAS/build_mesh.py

```diff
@@ -23,6 +24,11 @@ def removeFile(fileName):
         pass
 
 
+parser = argparse.ArgumentParser()
+parser.add_argument('--inject_cullCell',   default=0.0, type=float)
```

Comment:
@sbrus89, should this be renamed to something like `floodplain` with the scalar `floodplainElev` for clarity?

Reply:
Agreed, this has been addressed in the latest commit.